packages/react-styled-core/src/Input/index.js

```diff
@@ -56,7 +38,10 @@ const Input = forwardRef((
     <InputBase
       ref={ref}
       as="input"
-      css={css}
+      css={[ // TODO: combine the css array in v11
+        getInputGroupCSS({ variant }),
+        css,
+      ]}
```

Comment:
@tinaClin I finally decided to use this name within the `Input` component. I also found it's not necessary to use spread operator here to expand the object.